src/adapter.cpp

```diff
@@ -553,7 +553,7 @@ void dump_pids(int aid) {
     if (!p)
         return;
     for (i = 0; i < MAX_PIDS; i++)
-        if (p->pids[i].flags > 0) {
+        if (p->pids[i].flags > PID_STATE_INACTIVE) {
```

Comment:
This now includes PID_STATE_DELETED too, is that the idea? Or should it only concern active and new PIDs?

Reply:
Yeah this prints all pids and all states (if not 0)